cmd/istio-cni/redirect.go

```diff
@@ -64,6 +63,11 @@ var (
 	}
 )
 
+// Tables defines interface which must be implemented by tables mechanism driver (ex. ip tables )
+type Tables interface {
+	program(string, *Redirect) error
```

Comment:
Give names to arguments. I have no idea what the first string is about.

Reply:
done